src/main/java/eiteam/esteemedinnovation/commons/util/ReflectionHelper.java

```diff
@@ -23,7 +23,7 @@ public static Field getField(String fieldName, String obfName, Class clazz) {
         try {
             field = clazz.getDeclaredField(obfName);
         } catch (NoSuchFieldException e) {
-            FMLLog.warning("[EI] Unable to find field " + fieldName + " with its obfuscated " +
+            EsteemedInnovation.logger.warn("[EI] Unable to find field " + fieldName + " with its obfuscated " +
```

Comment:
`[EI]` should be removed from this, because it says `[esteemedinnovation]` in each log entry